examples/zephyr/stream/README.md

```diff
@@ -1,11 +1,12 @@
-# Golioth LightDB Stream sample
+# Golioth Stream service sample
 
 ## Overview
 
-This LightDB Stream application demonstrates how to connect with Golioth
-and periodically send data to LightDB Stream. In this sample temperature
-measurements are sent to `/temp` LightDB Stream path. For platforms that
-do not have temperature sensor a value is generated from 20 up to 30.
+This Golioth Stream application demonstrates how to connect with Golioth
+and periodically send data to the Stream service. In this sample
```

Comment:
Are we using "Stream" as a product name? Or just to describe a kind of data (i.e. "stream data")?

Reply:
@sam-golioth just to describe a kind of data, so I think saying `Golioth data streaming application` here may be a bit easier to understand 👍🏻